tutorials/tutorial1.md

```diff
@@ -0,0 +1,217 @@
+
+We train an AtacWorks model to learn a mapping from the 50-cell ATAC-seq signals to the 2400-cell ATAC-seq signal and peak calls. In other words, given a noisy ATAC-seq signal from 50 cells, this model learns what the signal would look like - and where the peaks would be called - if we had sequenced 2400 cells.
+
+## Step 1: Set parameters
+
+Replace 'path_to_atacworks' with the path to your cloned 'AtacWorks' github repository.
+```
+atacworks=<path_to_atacworks>
+```
+
+## Step 2: Download data
+
+We will download from AWS all of the data needed for this experiment. (Note: the S3 bucket is not yet public. We can use wget for download once it is.)
+
+### Noisy ATAC-seq signal from 50 Monocytes
+```
+aws s3 cp s3://atacworks-paper/dsc_atac_blood_cell_denoising_experiments/train_data/noisy_data/dsc.1.Mono.50.cutsites.smoothed.200.bw ./
+ ```
+### Clean ATAC-seq signal from 2400 Monocytes
+```
+aws s3 cp s3://atacworks-paper/dsc_atac_blood_cell_denoising_experiments/train_data/clean_data/dsc.Mono.2400.cutsites.smoothed.200.bw ./
+ ```
+### Clean ATAC-seq peaks from 2400 Monocytes
+```
+aws s3 cp s3://atacworks-paper/dsc_atac_blood_cell_denoising_experiments/train_data/clean_data/dsc.Mono.2400.cutsites.smoothed.200.3.narrowPeak ./
+```
+### Genomic intervals to define regions for training and validation
+
+We need to define which regions of the genome will be used to train and test the model. We want to train models on some portion of the genome ('training set') and evaluate their performance on a separate portion ('validation set'). We will choose the model that performs best on the validation set as the best model. Later, we will evaluate the performance of this best model on a third portion of the genome ('holdout set').
+
+Here, we use chromosome 20 as the validation set, chromosome 10 as the holdout set, and the remaining autosomes as the training set. Since a whole chromosome is too long to feed into the model at once, we split each of these chromosomes into 50,000-base long intervals. 
+
+We will download the intervals for the training set.
+```
+aws s3 cp s3://atacworks-paper/dsc_atac_blood_cell_denoising_experiments/intervals/hg19.50000.training_intervals.bed ./intervals/hg19.50000.training_intervals.bed
+```
+We can look at these intervals:
+
+```
+# head intervals/hg19.50000.training_intervals.bed 
+chr1  0 50000
+chr1  50000 100000
+chr1  100000  150000
+chr1  150000  200000
+chr1  200000  250000
+chr1  250000  300000
+chr1  300000  350000
+chr1  350000  400000
+chr1  400000  450000
+chr1  450000  500000
+```
+Next, we download the intervals for the validation set:
+```
+aws s3 cp s3://atacworks-paper/dsc_atac_blood_cell_denoising_experiments/intervals/hg19.50000.val_intervals.bed ./intervals/hg19.50000.val_intervals.bed
```

Comment:
Thought about this and I like the current version more. There are a lot of files in the folder, most of which are not useful here. Downloading them one at a time allows me to easily point to and describe each file.

Reply:
Are these files large/take a long time to download? If so, I would suggest adding a script that downloads the necessary files up front so the user can run it once and get everything instead of having to download and wait after each step. The tutorial can then explain each of the files as their usage comes up.

If the download time is very small, then it doesn't matter if it's downloaded up front or during the tutorial.

Reply:
downloads are not very big. I timed them, the biggest file takes about 40 secs or so.

Reply:
yes, all the downloads are very fast. Nothing should take more than a minute.

Reply:
Okay in that case I think it's good the way it is